projects/plugins/jetpack/modules/custom-post-types/nova.php

```diff
@@ -1587,6 +1587,10 @@ public function start_menu_item_loop( $query ) {
 	public function menu_item_loop_each_post( $post ) {
 		$this->menu_item_loop_current_term = $this->get_menu_item_menu_leaf( $post->ID );
 
+		if ( false === $this->menu_item_loop_current_term ) {
```

Comment:
Since `get_menu_item_menu_leaf` can also return `WP_Error` or `null`, should we check for such use-cases as well?

Reply:
Sure.  Updated the diff.